packages/dynamics-lib/src/index.js

```diff
@@ -12,6 +12,7 @@ export * from './entities/transaction-journal.entity.js'
 export * from './entities/transaction-currency.entity.js'
 export * from './entities/pocl-file.entity.js'
 export * from './entities/pocl-staging-exception.entity.js'
+// export * from './entities/pocl-data-validation-error.entity.js'
```

Comment:
can this comment be removed?

Reply:
Ah, thank you! This should be uncommented